packages/web-react/src/components/Footer/README.md

```diff
@@ -0,0 +1,256 @@
+# Footer
+
+The Footer is a highly variable composition. It comes in several ready-to-use design variants, but can be customized to
+achieve your specific design goals.
+
+## Composition
+
+The Footer supports several ready-to-use building blocks:
+
+- [Navigation Links](#navigation-links)
+- [Product Logo](#product-logo)
+- [Social Media Links](#social-media-links)
+- [Language Switch](#language-switch)
+- [Secondary Links](#secondary-links)
+
+This is how all supported building blocks of the Footer build up the complete composition:
+
+```jsx
+<Footer UNSAFE_className="bg-cover pt-1100 pb-1000">
```

Comment:
Once we have the `Footer` component, I'd move the classes into the internal implementation so the usage is as simple as this:

```suggestion
<Footer>
```